data/Ankur6702.json

```
@@ -0,0 +1,66 @@
+{
+  "name": "Ankur Agarwal",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "MERN Stack Developer | Google DSC Lead | Student Placement Coordinator | IIIT Kota",
+  "avatar": "https://drive.google.com/file/d/1fqSNRfbD8j9xUNJ7t-1ZdDeEwyIgcM6n/view?usp=sharing",
```

Please fix your avatar link to a more dynamic URL (avatar will get updated automatically when you change your GitHub Profile) + it's too short and clean! 😄

```suggestion
  "avatar": "https://github.com/Ankur6702.png",
```